File: retroravelry/src/main/java/com/caseykulm/retroravelry/models/user/SmallUser.java
package com.caseykulm.retroravelry.models.user;

import com.google.auto.value.AutoValue;

@AutoValue public abstract class SmallUser {

  public abstract int id();
  public abstract String large_photo_url();
  public abstract String photo_url();
  public abstract String small_photo_url();
  public abstract String tiny_photo_url();
  public abstract String username();

}

File: retroravelry/src/main/java/com/caseykulm/retroravelry/models/user/SmallUser.kt
package com.caseykulm.retroravelry.models.user

data class SmallUser(
    val id: Int,
    val large_photo_url: String?,
    val photo_url: String?,
    val small_photo_url: String?,
    val tiny_photo_url: String?,
    val username: String?
)
